index.rst

```diff
@@ -11,12 +11,35 @@
 Welcome to Flux's documentation!
 ================================
 
+Flux is a flexible framework for resource management, built for your site.
+The framework consists of a suite of projects, tools, and libraries which may be used to build site-custom resource managers for High Performance Computing centers.
+
 .. toctree::
-   :maxdepth: 3
-   :caption: Contents
+   :maxdepth: 2
+   :caption: Overview
 
-   quickstart
    contributing
+   quickstart
+
+.. toctree::
+   :maxdepth: 1
+   :caption: User Relevant RFCs
+
+   rfc/spec_1
+   rfc/spec_2
+   rfc/spec_7
+   rfc/spec_9
+
+.. toctree::
+   :maxdepth: 1
+   :caption: API Reference
+
+   rfc/README.md
```

Comment:
I think we chatted about this at the flux doc-a-thon, but I forget what the outcome was.  Is it possible to change the way the name here (and in the sidebar) renders from just `rfc` to `Flux RFC Index`?